### DISCUSSION TITLE
i did something wrong in intro-to-github

### DISCUSSION THREAD

**Discussion by LMAO-armv8** [community/community#35872] [GitHub Education]
i was given a task in intro-to-github program where i had to create a branch and do a edit , github action has to verify but i mistakely made commit on main branch , and workflow fails 

can someone tell what to do 

https://github.com/Global-Classroom/intro-to-github-LMAO-armv8

**Comment by samus-aran** [comment]
Hi @LMAO-armv8 👋 

How did you get on? Unfortunately I'm unable to view your repo so we, and I'm sure other students, would love to know how (and if) you were able to progress with the course.
